src/Metrics.ts

```diff
@@ -6,7 +6,7 @@ dotenv.config();
 // Access the token value
 const githubToken = process.env.API_TOKEN;
 if (!githubToken) {
-    throw new Error('GITHUB_TOKEN is not defined in the .env file');
+    throw new Error('API_TOKEN is not defined in the .env file');
```

Comment:
Why was process.env.GITHUB_TOKEN changed to API_TOKEN?

Reply:
This was changed in #30, your comment was "very nice"

anyways, this is done b/c variable can't be prefixed with "GITHUB"